monai/transforms/croppad/dictionary.py

```diff
@@ -293,11 +359,36 @@ def __init__(
     def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
         d = dict(data)
         for key in self.key_iterator(d):
+            orig_size = d[key].shape[1:]
             d[key] = self.cropper(d[key])
+            self.push_transform(d, key, orig_size=orig_size)
         return d
 
+    def inverse(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        d = deepcopy(dict(data))
 
-class RandSpatialCropd(RandomizableTransform, MapTransform):
+        for key in self.key_iterator(d):
+            transform = self.get_most_recent_transform(d, key)
+            # Create inverse transform
+            orig_size = np.array(transform[InverseKeys.ORIG_SIZE.value])
+            current_size = np.array(d[key].shape[1:])
+            pad_to_start = np.floor((orig_size - current_size) / 2)
+            # in each direction, if original size is even and current size is odd, += 1
+            pad_to_start[np.logical_and(orig_size % 2 == 0, current_size % 2 == 1)] += 1
+            pad_to_end = orig_size - current_size - pad_to_start
+            pad = np.empty((2 * len(orig_size)), dtype=np.int32)
+            pad[0::2] = pad_to_start
+            pad[1::2] = pad_to_end
+            inverse_transform = BorderPad(pad.tolist())
```

Comment:
For crop transforms, maybe we can consider to add inverse_pad args if users don't want to use default value to pad back.

Reply:
Yeah we discussed this at the time to have kwargs for the inverse arguments. Would you be OK to add that as a future feature request?